Launcher/Classes/Constants.swift

```diff
@@ -9,6 +9,7 @@ enum Constants {
         static let installSimulatorOrPluginDevice = "Please install a simulator or plug-in your device.".localized
         static let useLocalBuild = "Skipping download. Use a local app version.".localized
         static let notCompatibleWithDeviceType = "not compatible with chosen device type.".localized
+        static let wrongDeviceSetup = "Please, provide your device IP and bundle identifier. It can be done under 'Configure Device' settings.\n".localized
```

Comment:
Why the `\n` here?

Reply:
heh, good question. Didn't have time to investigate. We just need a new line there, as the method that outputs doesn't do it for free. Do you think it will be better to add the \n in the code?

Reply:
Yeah, I think it would be a better fit in the code.

Reply:
Done